history/peter-toma.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
```

Comment:
Do we know where he studied?  He was born in Hungary, I have a feeling that he left in 1940, when he was 16.

Reply:
There is nothing about his education, unfortunately.